### DISCUSSION TITLE
Aún no se me asigna grupo en teams

### DISCUSSION THREAD

**Discussion by nathalieCarrasco** [Launch-X-Latam/MisionFrontEnd#216] [Q&A]
Pude acceder a gibhub pero en teams no tengo equipo signado ayuda porfavor

**Comment by doguedogue** [comment]
Hola compañera, primero abre una página de incognito y entra a office.com con tu cuenta de @innovaccion.mx

Luego pega este [link de la forma](https://forms.office.com/r/Ddr5fxJbNk) que proporcionaron en el [Onboarding](https://github.com/Launch-X-Latam/OnBoarding) 
### https://forms.office.com/r/Ddr5fxJbNk

<a href="https://forms.office.com/r/Ddr5fxJbNk">

![image](https://user-images.githubusercontent.com/23409026/196252910-2f30f2ca-d8a9-4f25-97f4-763d0f448a89.png)

</a>
